Podfile

```diff
@@ -23,7 +23,7 @@ def shared_with_networking_pods
     pod 'AFNetworking', '3.2.1'
     pod 'Alamofire', '4.7.2'
     pod 'wpxmlrpc', '0.8.3'
-    pod 'WordPressKit', :git => 'https://github.com/wordpress-mobile/WordPressKit-iOS.git', :commit => 'c2ae16bbc3fb759e0684fc9e5642d52390d00285'
+    pod 'WordPressKit', :git => 'https://github.com/pinar1234/WordPressKit-iOS.git', :branch => 'develop'
```

Comment:
I'm guessing this is just for development purposes, but I wanted to flag that this will need to be changed to point back to our repos before it can get merged so it won't get lost :)

Reply:
yes you are totally right